pkg/cloud-provider/cloudapi/aws/aws_crd_helpers.go

```diff
@@ -70,6 +72,7 @@ func ec2InstanceToVirtualMachineCRD(instance *ec2.Instance, namespace string) *v
 	cloudID := *instance.InstanceId
 	cloudNetwork := *instance.VpcId
 
-	return utils.GenerateVirtualMachineCRD(cloudID, cloudName, cloudID, namespace, cloudNetwork, cloudNetwork,
-		v1alpha1.VMState(*instance.State.Name), tags, networkInterfaces, providerType)
+	return utils.GenerateVirtualMachineCRD(cloudID, strings.ToLower(cloudName), strings.ToLower(cloudID), namespace,
+		strings.ToLower(cloudNetwork), cloudNetwork, v1alpha1.VMState(*instance.State.Name), tags, networkInterfaces,
```

Comment:
Any specific reason for lowering casing

Reply:
There was a mismatch w.r.t VM annotations and VMSelector in CES for Azure. Without this change, Azure VM Match will not work. To keep it consistent, I updated AWS as well.